packages/editor/src/client/routes/articleList.tsx

```diff
@@ -158,9 +161,7 @@ export function ArticleList() {
             <HeaderCell>{t('articles.overview.updated')}</HeaderCell>
             <Cell dataKey="modifiedAt">
               {({modifiedAt}: ArticleRefFragment) =>
-                `${new Date(modifiedAt).toDateString()} ${new Date(
-                  modifiedAt
-                ).toLocaleTimeString()}`
+                `${dateTimeWithSecondsLocalString(new Date(modifiedAt))}`
```

Comment:
I recommend to also use the i18next translation service to format the date. Might be a bit more work if it's just the date but it's cleaner this way.